### ISSUE TITLE
AttributeError: module 'packaging' has no attribute 'version'

### ISSUE THREAD

**Issue by hiro0141mi** [joeynmt/joeynmt#201]
I have this error

```
(base) utagawa@musigny1:~/joeynmt$ python3 -m joeynmt train configs/tanaka_corpus.yaml
Traceback (most recent call last):
  File "/home/utagawa/anaconda3/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/utagawa/anaconda3/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/net/nas4/data/home/utagawa/joeynmt/joeynmt/__main__.py", line 64, in <module>
    main()
  File "/net/nas4/data/home/utagawa/joeynmt/joeynmt/__main__.py", line 44, in main
    train(cfg_file=args.config_path, skip_test=args.skip_test)
  File "/net/nas4/data/home/utagawa/joeynmt/joeynmt/training.py", line 791, in train
    joeynmt_version = packaging.version.parse(make_logger(model_dir, mode="train"))
AttributeError: module 'packaging' has no attribute 'version'
```
Also, I can't learn the translator (a directory named "models" is created, but when I open that directory, no files are created).

package and version are as follows

- packaging                          21.2
- joeynmt                            2.1.0
- Python 3.8.8

**Comment by hiro0141mi** [comment]
I solved the problem by changing 
`import packaging` to `import packaging.version` in training.py.

**Comment by may-** [comment]
Hi @hiro0141mi 

Thank you for letting us know this. We've fixed it in #203.
